src/features/emojiMod.ts

```diff
@@ -28,6 +31,27 @@ type ReactionHandlers = {
   ) => void;
 };
 
+// Send at most 1 log per
+const sendModLog = debounce(
+  (channelInstance: TextChannel, logBody: string) =>
+    channelInstance.send(logBody),
+  1000,
+  true,
+);
+const handleReport = (
+  channelInstance: TextChannel,
+  reportedMessage: Message,
+  logBody: string,
+) => {
+  if (warningMessages[reportedMessage.id]) {
+    warningMessages[reportedMessage.id].edit(logBody);
+  } else {
+    sendModLog(channelInstance, logBody).then((warningMessage) => {
```

Comment:
oops, I need to make sure this updates the cached list. forgot to copy off after splitting it out from sending